Tasks/Baranovskaya/Task3/task3.py

```diff
@@ -0,0 +1,71 @@
+        min = int(c[1])  
+        if len(c[0]) != 2 or len(c[1]) != 2: 
+            print("Введенное время не соотвествует указанному формату, неверная длина строки.")
+            continue
+     else:
+            print("Введенное время не соотвествует указанному формату, значения часов и минут должны быть цифрами.")
+            continue
+     if hour > 23 or min > 59:
+          print("Введенное время не соотвествует указанному формату, часы должны варьироваться от 00 до 23, а минуты - от 00 до 59.")
+          continue
+   else:
+       print("Нажмите 1 или 2 для выбора режима работы программы. Нажмите 3 для выхода.\n")
+       continue
+   if min == 0:
+    d = {1: "один", 2: "два", 3: "три", 4: "четыре", 5: "пять", 6: "шесть", 7: "семь", 8: "восемь", 9: "девять", 10: "десять",
+    11: "одиннадцать", 12: "двеннадцать", 13: "один", 14: "два", 15: "три", 16: "четыре", 17: "пять", 18: "шесть", 19: "семь", 
+    20: "восемь", 21: "девять", 22: "десять",  23: "одиннадцать", 24: "двенадцать", 0: "двенадцать"}
+    if (hour >= 2 and hour <= 4) or (hour >= 14 and hour <= 16):
+         print(f"Сейчас {d.get(hour)} часа ровно.")
+    elif hour == 1 or hour == 13:
+         print(f"Сейчас час ровно.")
+    else:
+         print(f"Сейчас {d.get(hour)} часов ровно.")
+   elif (min < 30) or (min > 30 and min < 45):
+     m = {1: "одна", 2: "две", 3: "три", 4: "четыре", 5: "пять", 6: "шесть", 7: "семь", 8: "восемь", 9: "девять", 10: "десять",
+     11: "одиннадцать", 12: "двеннадцать", 13: "тринадцать", 14: "четырнадцать", 15: "пятнадцать", 16: "шестнадцать", 17: "семнадцать", 
+     18: "восемнадцать", 19: "девятнадцать", 20: "двадцать", 30: "тридцать", 40: "сорок"}
+     tens, below_ten = divmod(min, 10)
+     if min == 1 or min == 21 or min == 31 or min == 41:
+          print(f"Сейчас {m.get(tens*10,'')} {m.get(below_ten)} минута {h.get(hour+1)}.\n")
+     elif (min >= 2 and min <= 4) or (min >= 22 and min <= 24) or (min >= 32 and min <= 34) or (min >= 42 and min <= 44):
+          print(f"Сейчас {m.get(tens*10,'')} {m.get(below_ten)} минуты {h.get(hour+1)}.\n")
+     elif min >= 5 and min <= 19:
+          print(f"Сейчас {m.get(min)} минут {h.get(hour+1)}.\n")
+     else:
+          print(f"Сейчас {m.get(tens*10)} {m.get(below_ten, '')} минут {h.get(hour+1)}.\n")
+   elif min == 30:
+      print(f"Сейчас половина {h.get(hour+1)}.\n")
+   elif min >= 45:
+     d = {1: "час", 2: "два", 3: "три", 4: "четыре", 5: "пять", 6: "шесть", 7: "семь", 8: "восемь", 9: "девять", 10: "десять",
+     11: "одиннадцать", 12: "двеннадцать", 13: "час", 14: "два", 15: "три", 16: "четыре", 17: "пять", 18: "шесть", 19: "семь", 
+     20: "восемь", 21: "девять", 22: "десять",  23: "одиннадцать", 24: "двенадцать"}
+     m = {59: "одной", 58: "двух", 57: "трех", 56: "четырех", 55: "пяти", 54: "шести", 53: "семи", 52: "восьми", 51: "девяти", 50: "десяти",
+     49: "одиннадцати", 48: "двеннадцати", 47: "тринадцати", 46: "четырнадцати", 45: "пятнадцати"}
```

Comment:
I would suggest to find a way to combine those dictionaries into a singular data structure, it would make the algorithm more clear and straightforward